search-service/src/test/kotlin/com/egm/stellio/search/discovery/service/EntityTypeServiceTests.kt

```diff
@@ -15,9 +15,34 @@ import com.egm.stellio.search.support.WithTimescaleContainer
 import com.egm.stellio.search.support.gimmeEntityPayload
 import com.egm.stellio.shared.model.APIException
 import com.egm.stellio.shared.model.ResourceNotFoundException
-import com.egm.stellio.shared.util.*
+import com.egm.stellio.shared.util.APIARY_COMPACT_TYPE
+import com.egm.stellio.shared.util.APIARY_TYPE
+import com.egm.stellio.shared.util.APIC_COMPOUND_CONTEXTS
+import com.egm.stellio.shared.util.AQUAC_COMPOUND_CONTEXT
+import com.egm.stellio.shared.util.BEEHIVE_COMPACT_TYPE
+import com.egm.stellio.shared.util.BEEHIVE_TYPE
+import com.egm.stellio.shared.util.CATEGORY_COMPACT_VOCABPROPERTY
+import com.egm.stellio.shared.util.CATEGORY_VOCAPPROPERTY
+import com.egm.stellio.shared.util.FRIENDLYNAME_COMPACT_LANGUAGEPROPERTY
+import com.egm.stellio.shared.util.FRIENDLYNAME_LANGUAGEPROPERTY
+import com.egm.stellio.shared.util.INCOMING_COMPACT_PROPERTY
+import com.egm.stellio.shared.util.INCOMING_PROPERTY
 import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_LOCATION_PROPERTY
 import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_LOCATION_TERM
+import com.egm.stellio.shared.util.LUMINOSITY_COMPACT_JSONPROPERTY
+import com.egm.stellio.shared.util.LUMINOSITY_JSONPROPERTY
+import com.egm.stellio.shared.util.MANAGED_BY_COMPACT_RELATIONSHIP
+import com.egm.stellio.shared.util.MANAGED_BY_RELATIONSHIP
+import com.egm.stellio.shared.util.OUTGOING_COMPACT_PROPERTY
+import com.egm.stellio.shared.util.OUTGOING_PROPERTY
+import com.egm.stellio.shared.util.SENSOR_COMPACT_TYPE
+import com.egm.stellio.shared.util.SENSOR_TYPE
+import com.egm.stellio.shared.util.TEMPERATURE_PROPERTY
+import com.egm.stellio.shared.util.ngsiLdDateTime
+import com.egm.stellio.shared.util.shouldFail
+import com.egm.stellio.shared.util.shouldSucceedWith
+import com.egm.stellio.shared.util.toUri
+import com.egm.stellio.shared.util.typeNotFoundMessage
```

Comment:
I think this number of import can be an issue.
But i think we should solve it by putting the properties/terms/types in objects instead of removing the rule.